README.md

```diff
@@ -114,9 +114,9 @@ sccache supports gcc, clang, MSVC, rustc, [NVCC](https://docs.nvidia.com/cuda/cu
 
 If you don't [specify otherwise](#storage-options), sccache will use a local disk cache.
 
-sccache works using a client-server model, where the server runs locally on the same machine as the client. The client-server model allows the server to be more efficient by keeping some state in memory. The sccache command will spawn a server process if one is not already running, or you can run `sccache --start-server` to start the background server process without performing any compilation.
+sccache works using a client-server model, where the server runs locally on the same machine as the client. The client-server model allows the server to be more efficient by keeping some state in memory. The sccache command will spawn a server process if one is not already running, or you can run `sccache --start-server[=port]` to start the background server process without performing any compilation.
```

Comment:
I would prefer we don't have this syntax
`sccache --start-server[=port]`

Reply:
Any preferences?

Reply:
I can have it `sccache --start-server [port]` if that's any better

Reply:
why not --port ?

Reply:
Because I figure something like `--port` will only ever show up when you're starting or stopping the server, making it a subset of `--start-server` (or stop-server), I figure it's better to not have a dedicated `--port`, and just have `[port]` be an optional argument to `--start-server` (or stop, again) rather than have another optional flag. Because the usage would be the exact same in either case, except when they want to specify it, they just need to type more. Not a big reason, but I think that's more or less why, from my perspective.
What do you think?

Reply:
i am really not a fan of the --start-server[=port] syntax :/
@glandium WDYT ?